components/org.wso2.carbon.micro.integrator.security/src/main/java/org/wso2/carbon/micro/integrator/security/vault/Constants.java

```diff
@@ -49,6 +49,7 @@
     public static final String CIPHER_STANDALONE_CONFIG_PROPERTY_FILE = "cipher-standalone-config.properties";
     public static final String SECRET_PROPERTY_FILE_PROPERTY = "secret.conf.properties.file";
     public static final String CIPHER_TRANSFORMATION_SYSTEM_PROPERTY = "org.wso2.CipherTransformation";
+    public static final String SECURE_VALULT_PASSWORD_REGEX_SYSTEM_PROPERTY = "org.wso2.SecureVaultPasswordRegEx";
```

Comment:
Don't we need to add this to the carbon.properties file?

Reply:
Also IMO, this need to go to our docs?

Reply:
**org.wso2.SecureVaultPasswordRegEx=^[\\S]{5,30}$** property added to the carbon.properties file.